tasks/cron_jobs.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Removing old records in the repl_monitor table
+  cron:
+     name: "Removing repl_monitor records older than 4 days"
```

Comment:
This message will be for all users? Maybe need a simple message like 'Removing repl_monitor old records'